### ISSUE TITLE
[JOSS Review] Demo testing 

### ISSUE THREAD

**Issue by zhangfanmark** [DeepRegNet/DeepReg#478]
Hi, 

I'm reviewing the DeepReg paper on JOSS (openjournals/joss-reviews/issues/2705). I am running some testing examples here.

1. python3 demos/paired_ct_lung/demo_data.py

ERROR:
```
(deepreg) [fan@localhost DeepReg]$ python3 demos/paired_ct_lung/demo_data.py
2020-10-23 17:59:59.095778: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
Downloading data from https://zenodo.org/record/3835682/files/training.zip
278970368/278965344 [==============================] - 407s 1us/step
The file  training.zip  has successfully been downloaded!
Files unzipped!
Traceback (most recent call last):
  File "demos/paired_ct_lung/demo_data.py", line 164, in <module>
    destination = new_name
NameError: name 'new_name' is not defined
```

2.  python3 demos/paired_ct_lung/demo_data.py

README says "A pre-trained model will be downloaded", but the model seem not there.

3. It might be a good idea to have one master testing script to call the demo by its name.

**Comment by mathpluscode** [comment]
Hi @zhangfanmark , the mentioned demo has been fixed in #470. Now all demos have been tested. Please feel free to try again and let us know in case of any problems?

**Comment by mathpluscode** [comment]
Hi @zhangfanmark  thank you again for your review. I'm closing this as we are preparing for the acceptance and the release of `v0.1.0`.
